dashboard/app/models/script.rb

```diff
@@ -521,6 +525,10 @@ def hoc_finish_url
     end
   end
 
+  def csf_finish_url
+    CDO.code_org_url "/api/csf/finish/#{name}"
+  end
```

Comment:
This seems like an unnecessary trip to the server. Why not send them directly to /congrats here?